modules/artifact/src/test/java/org/mule/runtime/module/artifact/builder/AbstractDependencyFileBuilder.java

```diff
@@ -82,6 +88,11 @@ public File getArtifactPomFile() {
       model.setArtifactId(getArtifactId());
       model.setVersion(getVersion());
       model.setModelVersion("4.0.0");
+      if (!sharedLibraries.isEmpty()) {
+        model.setBuild(new Build());
+        model.getBuild().setPlugins(new ArrayList<>());
```

Comment:
change to `model.getBuild().setPlugins(singletonList(createMuleMavenPlugin()));`?